plugins/mac-avcapture/AVCaptureDeviceFormat+OBSListable.m

```diff
@@ -77,13 +77,15 @@ - (NSNumber *)aspectRatioComparisonValue
         CMFormatDescriptionRef formatDescription = self.formatDescription;
         CMVideoDimensions dimensions = CMVideoFormatDescriptionGetDimensions(formatDescription);
 
-        //for sorting purposes, treat aspect ratios sufficiently close together as equivalent
-        double ratio = (double) dimensions.width / (double) dimensions.height;
-        double roundedRatio = round(ratio * 100) / 100;
-        NSNumber *compareRatio = @(roundedRatio);
+        if (dimensions.height != 0) {
```

Comment:
This would evaluate to true if `dimensions.height` is `-500` - is that desired?

Reply:
I'm not sure. My priority was to avoid a crash if a card advertises some nonsense format with zero in some dimension and that squeaks past AVFoundation.

It's may be more 'correct'/desired to just not assign an aspect ratio sort value to those hypothetical formats with negative dimensions, I suppose? I would have to double check that an NSComparison between an object and nil doesn't raise an exception in that case too.

Reply:
Changed to just not set a value for anything zero or negative, and also changed the NSString representation to return `@""` if either dimension is <= 0. Verified that nil values for the aspect ratio do not crash if used in a sort descriptor.